backend.go

```diff
@@ -115,6 +149,31 @@ func (b *kubeAuthBackend) loadConfig(ctx context.Context, s logical.Storage) (*k
 	if config == nil {
 		return nil, errors.New("could not load backend configuration")
 	}
+
+	// Nothing more to do if loading local CA cert and JWT token is disabled.
+	if config.DisableLocalCAJwt {
+		return config, nil
+	}
+
+	// Read local JWT token unless it was not stored in config.
+	if config.TokenReviewerJWT == "" {
+		if b.localSATokenReader == nil {
+			b.localSATokenReader = newCachingFileReader(b.localJWTPath, jwtReloadPeriod, b.currentTime)
+		}
+		config.TokenReviewerJWT, _ = b.localSATokenReader.ReadFile()
+		// Ignore error: make best effort trying to load local JWT,
+		// otherwise the JWT submitted in login payload will be used.
+	}
+
+	// Read local CA cert unless it was stored in config.
+	if config.CACert == "" {
+		buf, err := ioutil.ReadFile(b.localCACertPath)
+		if err != nil {
+			return nil, err
+		}
+		config.CACert = string(buf)
```

Comment:
Good catch, my mistake! I previously loaded CA cert when this backend was initialized in `InitializeFunc` but when I removed it due to previous fixes, I accidentally also removed keeping CA cert in memory.  

I've now added reading CA cert into `kubeAuthBackend` but only once in first login (not reloaded periodically).  It could of course use the same `cachingFileReader` but I think it is rare for the clients to support rotating CA cert.